pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -17,16 +17,27 @@
 """
 
 # pylint: disable=too-many-arguments
+import semantic_version
 import jax
 import jax.numpy as jnp
 
 import pennylane as qml
 from pennylane.interfaces.jax import _compute_jvps
-from pennylane.interfaces.jax_jit import _numeric_type_to_dtype, _validate_jax_version
+from pennylane.interfaces.jax_jit import _numeric_type_to_dtype
+from pennylane.interfaces import InterfaceUnsupportedError
 
 dtype = jnp.float64
 
 
+def _validate_jax_version():
```

Comment:
Why is there a copy of this function in this file? The only difference I notice is the word "new" in the message.

Reply:
I run into circular import issue if I import from the file